apps/cli/README.md

```diff
@@ -119,6 +119,44 @@ After creating a new project and installing the dependencies, open [http://local
 
 Start editing the page by modifying `app/page.tsx` or `pages/index.ts`, as per your NextJS config.
 
+## Contract
+
+### Modifying Contract Code
+
+Find the atomic asset contract code in the `src/contracts` directory of the created project. You can make necessary modifications to the codebase according to your needs.
+
+### Deploying Contract Source
+
+To deploy the contract source, follow the commands provided below.
+
+#### With Wallet Keyfile Present at Root
+
+Please ensure that the wallet keyfile `wallet.json` is present at the root of the project.
+
+```bash
+npm run deploy-contracts
+# or
+yarn deploy-contracts
+# or
+pnpm deploy-contracts
+# or
+bun deploy-contracts
+```
+
+#### Without Wallet Keyfile at Root
+
+If the wallet keyfile is not present at the root of the project, you can specify its path while deploying the contracts:
+
+```bash
+npm run deploy-contracts /Users/arweave/Documents/keys/wallet.json
+# or
+yarn deploy-contracts /Users/arweave/Documents/keys/wallet.json
+# or
+pnpm deploy-contracts /Users/arweave/Documents/keys/wallet.json
+# or
+bun deploy-contracts /Users/arweave/Documents/keys/wallet.json
+```
+
```

Comment:
Is this too much information for a README? Can we instead add a simply line in the "What are the key components section" or in the "Preview" saying 

"The project template is initialized with a default contract whose logic is located in `path/to/logic`. For replacing the template contract with a custom one, simply edit the logic and run the script `deploy-contracts`. This will automatically update the contract linked functionality to the new one."

Reply:
Maybe the commands section is unnecessary and we can write for only one package manager as an example. I can reduce the content written for the contract section.

Reply:
Yeah sounds good